src/App.js

```diff
@@ -1,29 +1,95 @@
-import logo from './logo.svg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import { useState } from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import ProductList from "./pages/ProductList";
+import Cart from "./pages/Cart";
+import ProductDetail from "./pages/ProductDetail";
+import Store from "./contexts/Store";
+import Checkout from "./pages/Checkout";
 
-function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
+const App = () => {
+  const [cart, setCart] = useState([]);
 
+  const updateCart = (cart, product, numItems) => {
+    let newCart = JSON.parse(JSON.stringify(cart));
+    let item = [];
+    if (Object.keys(product).length !== 0) {
+      if (newCart.length > 0) {
+        item = newCart.filter((cartItem) => cartItem.id === product.id);
+      }
```

Comment:
to initialize this variable the ternary is better to read intend of the LET+IF

```suggestion
    const item = cart.length > 0
      ? cart.filter((cartItem) => cartItem.id === product.id)
      : [];
    if (Object.keys(product).length !== 0) {
```